src/lib/djerba/configure.py

```diff
@@ -98,6 +98,7 @@ def discover_primary(self):
             self.logger.info("Searching provenance for workflow output files")
             updates[ini.SEQUENZA_FILE] = self.reader.parse_sequenza_path()
             updates[ini.MAF_FILE] = self.reader.parse_maf_path()
+            updates[ini.MSI_FILE] = self.reader.parse_msi_path()
```

Comment:
That sounds good, I think an optional input constant is still a good idea. I think we'll want to chat about potentially running MSISENSOR for old cases eventually? Is this too much of a hassle? It seems like it would be a good thing to include in all reports not just new reports?

Reply:
I think running the workflow on older samples would just be a matter of switching on the olive for the relevant projects, so not too bad. We can certainly ask for it if the data would be useful. But I think we still want to be able to generate Djerba reports without it, particularly for testing and GSICAPBENCH reports.